core-build/gulp-core-build/src/GulpProxy.ts

```diff
@@ -1,32 +0,0 @@
-// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
-// See LICENSE in the project root for license information.
-
-import gulp = require('gulp');
-import Orchestrator = require('orchestrator');
-
-/* tslint:disable:max-line-length */
-/**
- * A helper utility for gulp which can be extended to provide additional features to gulp vinyl streams
- */
-export class GulpProxy extends Orchestrator implements gulp.Gulp {
```

Comment:
>GulpProxy [](start = 13, length = 9)

What is this change?

Reply:
I removed gulpproxy because it was causing a lot of issues with node/gulp typings, but I may be able to revert this as well

---
In reply to: [159794855](https://github.com/Microsoft/web-build-tools/pull/383#discussion_r159794855) [](ancestors = 159794855)

Reply:
I think this can also be reverted due to the deduping change

---
In reply to: [159795595](https://github.com/Microsoft/web-build-tools/pull/383#discussion_r159795595) [](ancestors = 159795595,159794855)